adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -305,6 +307,10 @@ public boolean isExpired() {
         return TokenCacheItem.isTokenExpired(getExpiresOn());
     }
 
+    public final String getAuthority() {
```

Comment:
>getAuthority [](start = 24, length = 12)

I didn't figure out what's changed here, getAuthority returns mCloudInstanceName is still confusing :)

Reply:
I changed the method name without changing the underlying variable, now fixed.